config/beaker_hosts/perf-test.cfg

```diff
@@ -0,0 +1,55 @@
+default_vmname: &default_vmname centos-7-x86_64
+default_platform: &default_platform el-7-x86_64
+default_snapshot: &default_snapshot pe
+
+:root_keys: true
+
+:answers:
+  :console_admin_password: puppetlabs
+:department: eso-dept
```

Comment:
We could just wait for the refactor... but since you were messing with the file I figured I would at least point out that most of this file does nothing.  All the abs stuff is meaningless.  The abs_helper handles all those settings.  So like department, lifetime, volume size...  I am not even sure this file does anything for an abs based run.  I believe it does have meaning for vmpooler acceptance testing though.

Reply:
I would suggest to wait for the refactor. This file 'works out of the box', so cleaning it up here is not in alignment with the objectives of the PR.